pkg/visibility/api/rest/pending_workloads_lq_test.go

```diff
@@ -0,0 +1,406 @@
+// Copyright 2023 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rest
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/endpoints/request"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	visibility "sigs.k8s.io/kueue/apis/visibility/v1alpha1"
+	"sigs.k8s.io/kueue/pkg/constants"
+	"sigs.k8s.io/kueue/pkg/queue"
+	utiltesting "sigs.k8s.io/kueue/pkg/util/testing"
+)
+
+type req struct {
+	nsName      string
+	queueName   string
+	queryParams *visibility.PendingWorkloadOptions
+}
+
+type resp struct {
+	wantErr              error
+	wantPendingWorkloads []visibility.PendingWorkload
+}
+
+func TestPendingWorkloadsInLQ(t *testing.T) {
+	const (
+		nsNameA  = "nsA"
+		nsNameB  = "nsB"
+		cqNameA  = "cqA"
+		cqNameB  = "cqB"
+		lqNameA  = "lqA"
+		lqNameB  = "lqB"
+		lowPrio  = 50
+		highPrio = 100
+	)
+
+	var (
+		defaultQueryParams = &visibility.PendingWorkloadOptions{
```

Comment:
nit: I think no need to have to wrap this in `var`?Maybe just `defaultQueryParams := `?